docs/docs/guides/basics/eth.md

```diff
@@ -344,6 +343,359 @@ Transaction receipt: {
 }
 ```
 
+## Importing specific package
```

Comment:
I think, starting from here, the following needs to be sperate from the above 4 steps tutorial. Either put those in a new file. Or if you like you can rename the old file. And add a new file with the same name of the old one. However, the important thing is to keep the new additional docs seperated from the above tutorial, or not?